lib/opencensus/trace/integrations/sidekiq_middleware.rb

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "opencensus"
+require "opencensus/trace/integrations/sidekiq"
+require "opencensus/trace/integrations/active_support"
```

Comment:
A colleague pointed out that if we remove this `require` statement we enable Non-Rails Sidekiq users to use this middleware.

TODO: remove dependency on `active_support` integration in `sidekiq_middleware.rb`

Reply:
👍

Reply:
I removed the require and added it to the documentation